node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -138,84 +136,57 @@ fn collect_externals_from_multi_config(
 }
 
 fn extract_values_vcl_fill_multiconfig_vec(
-    config_file_vcl: Box<dyn VirtualCommandLine>,
-    environment_vcl: Box<dyn VirtualCommandLine>,
-    commandline_vcl: Box<dyn VirtualCommandLine>,
-    user_specific_data: UserSpecifiedData,
+    multi_config_vec: Vec<Box<dyn VirtualCommandLine>>,
+    initialization_data: ConfigInitializationData,
+    app: &App,
 ) -> (Vec<String>, Vec<String>) {
+    let full_multi_config =
```

Comment:
Should this be passed in as a single parameter, rather than constructed on the spot like this? It seems that it would make this function and its caller more loosely coupled, since this function wouldn't have to know about the `multi_config_vec` and the `app`, only the `MultiConfig` itself, which is already widely known.